versions/development/parsers/antlr4/Python3/README.md

```diff
@@ -0,0 +1,28 @@
+# WDL Python2 Parser
+
+This directory provides a parser implemented in Python3. The parser is built on top of the Base [WDLBaseLexer.py](WdlParser/WDLBaseLexer.py)
+
+# Requirements
+- python3
+
+# Installing the runtime
+
+```bash
+pip install antlr4-python2-runtime
```

Comment:
Again python2 instead of python3

As a more general comment: Python2 is going to be deprecated within two months.
It might be better to remove the python2 parser all together.